src/cocoa/toga_cocoa/widgets/button.py

```diff
@@ -2,7 +2,7 @@
 from travertino.size import at_least
 
 from toga_cocoa.libs import *
```

Comment:
At column 1: (F403) 'from toga_cocoa.libs import *' used; unable to detect undefined names